src/transformers/models/marian/modeling_marian.py

```diff
@@ -1301,7 +1301,7 @@ def prepare_inputs_for_generation(
             "use_cache": use_cache,  # change this to avoid caching (presumably for debugging)
         }
 
-    def adjust_logits_during_generation(self, logits, cur_len, max_length):
+    def adjust_logits_during_generation(self, logits, cur_len, max_length, **kwargs):
```

Comment:
don't need this function anymore no?

Reply:
oh Marian is special here in that in never predicts the pad token -> we should handle this with the `bad_token_ids` generation param though and delete the function here as well